code/nuxeo-platform-rendition-web/src/main/java/org/nuxeo/ecm/platform/rendition/url/RenditionBasedCodec.java

```diff
@@ -71,11 +71,11 @@
     public static final String PATH_URL_PATTERN = "/" // slash
             + "([\\w\\.]+)" // server name (group 1)
             + "(?:/(.*))?" // path (group 2) (optional)
-            + "@([\\w\\-\\.\\%]+)" // renditionName (group 3)
+            + "@([\\w\\-\\.\\%\\,]+)" // renditionName (group 3)
```

Comment:
Do you really need to escape the comma?

Reply:
comma escaped  only in  renditionName  of the URL_PATTERN allow to fix the issue

Reply:
OK correction done, indeed comma doesn't need to be escaped (with \\)  in a java Regex expression